tools/gulpmain.ts

```diff
@@ -64,7 +64,14 @@ if (options.modulesFilter) {
 }
 
 /* Default task for building /src folder into /bin */
-gulp.task("default", ["install-build"]);
+gulp.task("default", [], function (callback: gulp.TaskCallback): void {
+    var os = require ("os");
+    if (os.platform() === "win32") {
+        runSequence("install-build", "link", callback);
```

Comment:
This could still fail if someone has their npm prefix pointing to program files, which I've seen sometimes. Not as common though.